server-spi-private/src/main/java/org/keycloak/models/utils/RepresentationToModel.java

```diff
@@ -187,6 +188,7 @@ public static void importRealm(KeycloakSession session, RealmRepresentation rep,
         if (rep.getNotBefore() != null) newRealm.setNotBefore(rep.getNotBefore());
 
         if (rep.getDefaultSignatureAlgorithm() != null) newRealm.setDefaultSignatureAlgorithm(rep.getDefaultSignatureAlgorithm());
+        else newRealm.setDefaultSignatureAlgorithm(Algorithm.RS256);
```

Comment:
I suppose that instead of have the default algorithm hardcoded here, it will be better to use some constant and use it from all the places? Basically when we later decide that default algorithm should be changed to for example RS512, we will need to change it in single place and not on many random places inside codebase (which would mean possible bugs as of course some places will be forgotten).

Also will be good to have a test for this.